test/optimove_flutter_sdk_method_channel_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/services.dart';
```

Comment:
Do we plan to implement any tests, or should we remove this?

Reply:
Removed it for now